midend/simplifyKey.h

```diff
@@ -42,7 +46,9 @@ class NonLeftValue : public KeyIsComplex {
     bool isTooComplex(const IR::Expression* expression) const;
```

Comment:
The pass applies to non-lvalue expressions. All lvalues are considered not complex and therefore not transformed.